ee/clickhouse/views/actions.py

```diff
@@ -90,10 +92,13 @@ def count(self, request: Request, **kwargs) -> Response:  # type: ignore
         if query == "":
             return Response({"count": 0})
 
-        results = sync_execute(
-            "SELECT count(1) FROM events WHERE team_id = %(team_id)s AND {}".format(query),
-            {"team_id": action.team_id, **params},
-        )
+        try:
+            results = sync_execute(
+                "SELECT count(1) FROM events WHERE team_id = %(team_id)s AND {}".format(query),
+                {"team_id": action.team_id, **params},
+            )
+        except CHQueryErrorCannotCompileRegexp:
```

Comment:
Related to comment above, but why do we raise a Regexp compile error here? Is the only error that happen here related to a regex error?

Reply:
There can be several errors when querying CH (see the file above where I define this for the rest of them - all 500 of them), and this on is specific to the user input being provided for selectors / regexes.

Reply:
> Is the only error that happen here related to a regex error? 

Yes